aicsimageio/metadata/README.md

```diff
@@ -0,0 +1,31 @@
+# AICSImage Metadata Module
+
+Undecided how to organize this module. This likely isn't included in 4.0 but more
+likely 4.1 or similar but good to start thinking about.
+
+I am currently thinking:
+```
+aicsimageio/
+    metadata/
+        ome/
```

Comment:
proposal: remove all other subdirectories and make OME our only metadata format.  
aicsimageio/metadata/[omexml.py, transform.py, xslt/,...]
We can read other formats into omexml but can only write out omexml.  Until further notice or the new uber-metadata format spec comes out.

Reply:
Also, xslt may not be the only means of transforming, right?  xslt is only xml-->xml as far as I know (correct me if i'm wrong).  There may be json-->xml in some cases.